### PR TITLE
automatic system detection for mac & linux, add setup script

### PR DESCRIPTION
**Pull request by Dziqus**

### CODE REVIEW

File: installers/mac/macos-dev.sh [hunk 1]
```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
-# turn of akward mouse acceleration: otherwise host is hard to handle using VNC mouse
+# to run this script use: bash <(curl -Ls -w %{url_effective} https://tinyurl.com/vhhm3zz)
+
+# turn of akward mouse acceleartion: otherwise host is hard to handle using VNC mouse
 defaults write .GlobalPreferences com.apple.mouse.scaling -1
 
+# enable ssh
+sudo launchctl load -w /System/Library/LaunchDaemons/ssh.plist
```
Comment (ivy-rew) [inline installers/mac/macos-dev.sh]: nice, thanks for sharing

File: bin/newDesigner.sh [hunk 2]
```diff
@@ -14,6 +14,17 @@ if [ ! -z "$2" ]
     JOB=$2
 fi
 
+unameOut="$(uname -s)"
```
Comment (ivy-rew) [inline bin/newDesigner.sh]: please come with an another PR for this change... 
i see the mac-dev.sh as completed - but this has not much to do with the other changes

Comment (ivy-rew) [reply bin/newDesigner.sh]: what I'd like to change here: can you isolate that code in a function and only execute it if .env template has no value set in this variable? or even remove it completely from .env (if we can determine the correct version also for windows)

Comment (Dziqus) [reply bin/newDesigner.sh]: sure will do :)

**Pull request merged**